public/js/data.js

```diff
@@ -1007,8 +990,8 @@ function generateImage(imgType, view, transparent, resolution, down = true) {
 
 }
 
-if (logix_project_id == 0)
-setTimeout(promptSave,120000);
+// if (logix_project_id == 0)
+// setTimeout(promptSave,120000);
```

Comment:
> 
> 
> I tested, it is working on the whole. It is working on Mac as well. There are some issues:
> 
>     1. Delete - backspace key is no longer working. Note that backspace is delete on mac.
> 
>     2. CMD/Ctr + R is not refreshing page
> 
>     3. Copy/Paste is no longer working
> 
>     4. Check other regular shortcuts
> 
> 
> Don't do preventDefault or stopPropagate if that shortcut is not mapped. Looks like many browser shortcuts are disabled. Like Ctr + L should select URL of the current tab.
> 
> CMD is working on Mac as required. I found two files which have `value.replace("Ctrl", "Meta")`, do we need to have them in 2 places?
> 
> Also it will be helpful if you could tell which files are written by you and which files are from the library.

I'll rectify these bugs soon.

& I the plugin files are named as xxxx.plugin.js, there are currently 2 of them in the hotkey implementation, shortcuts.plugin.js under modal dir & normalize.plugin.js under modal/normalize dir, rests are written by me.

Reply:
> 
> 
> Why is this commented out?

I comment them out every time I start working on the simulator for a smooth workflow, this time I forgot to uncomment it, I shall do it in the next commit.